bridge.go

```diff
@@ -353,6 +353,8 @@ func (b *bridge) SyncSourceFor(id ProjectIdentifier) error {
 	return b.sm.SyncSourceFor(id)
 }
 
+func (b *bridge) Release() { b.sm.Release() }
```

Comment:
Let's have this panic, instead. This should never be called on a bridge; by definition, the bridge only exists for the duration of a solve run, and a solve run is never responsible for closing the source manager it's using. Having a panic here is the best way to make that explicit.

Also, we'll be able to back this out soon, because I plan to have fixing #159 include dropping the composition of `SourceManager` into `sourceBridge`.